phpstan.neon

```diff
@@ -103,6 +107,7 @@ parameters:
     - '#Parameter \#1 \$callback of method Illuminate\\Support\\Collection<string,mixed>::sortBy\(\) expects array<array{string, string}\|\(callable\(mixed, string\): mixed\)\|string>\|\(callable\(mixed, string\): mixed\)\|string, Closure\(string, string\): int given\.#'
     - '#Parameter \#1 \$callback of method Laravel\\Nova\\Actions\\Action::canRun\(\) expects Closure\(Laravel\\Nova\\Http\\Requests\\NovaRequest, mixed\): bool, Closure\(Laravel\\Nova\\Http\\Requests\\NovaRequest, App\\Models\\[a-zA-Z]+\): bool given\.#'
     - '#Parameter \#1 \$channel of method Illuminate\\Notifications\\Messages\\SlackMessage::to\(\) expects string, string\|null given\.#'
+    - '#Parameter \#1 \$column of method Illuminate\\Database\\Eloquent\\Builder<App\\Models\\Payment>::where\(\) expects array<int\|string, mixed>\|\(Closure\(Illuminate\\Database\\Eloquent\\Builder<App\\Models\\Payment>\): void\)\|\(Closure\(Illuminate\\Database\\Eloquent\\Builder<App\\Models\\Payment>\): Illuminate\\Database\\Eloquent\\Builder<App\\Models\\Payment>\)\|Illuminate\\Database\\Query\\Expression\|string, Closure\(Illuminate\\Database\\Eloquent\\Builder\): Illuminate\\Database\\Eloquent\\Builder<Illuminate\\Database\\Eloquent\\Model> given\.#'
```

Comment:
It seems unusual to have to ignore this error but what it's referencing seems like a perfectly valid use of `where` to me:

https://github.com/RoboJackets/apiary/blob/8f02f6ecd3b5c42f17a6a69087bc81d260ec3dad/app/Http/Controllers/PaymentController.php#L70

Reply:
That says the closure should return `void`. I feel like that would be more idiomatic, though also definitely more verbose than using arrow functions.